### ISSUE TITLE
[QUESTION] Misunderstanding about accuracies

### ISSUE THREAD

**Issue by azman0101** [tdeekens/promster#26]
According to [Prometheus Best Practices](https://prometheus.io/docs/practices/naming/#base-units), unit for time is "seconds".

To mesure duration under the second, we need to use float with suffisant level of precision.

The choice to implement different units, each one of them, mapped to a specific level of precision make impossible to follow Prometheus Best Practices.

What do you think about de-couple precision and unit ?

**Comment by tdeekens** [comment]
Thanks for the issue. Yeah, seconds is a Prometheus best practice. Sadly I only learned after the initial version with milliseconds. Furthermore, loads of Hapi/Express middlewars have milliseconds so I wanted to offer some sort of compatibility.

> mapped to a specific level of precision make impossible to follow Prometheus Best Practices.

I am not sure I am able to follow, sorry. In case of seconds being chosen as one (or the only) accuracy the metric name will have the `_seconds` suffix which to my understanding is also a best practice. Furthermore the unit of the metric value will be whatever the `prom-client` decides to respond with after being instrumented with values [here](https://github.com/tdeekens/promster/blob/master/packages/metrics/modules/create-request-observer/create-request-observer.js#L68).

**Comment by azman0101** [comment]
I'll try to give you example of what I mean:

When I query promster generated metric `http_request_duration_seconds_bucket{namespace="xx-xx-440"} < 1` responses are only 0s. However, I know the response is between 0 and 1... So this make me conclude that the reponse is rounded.

For the query `nginx_ingress_controller_ingress_upstream_latency_seconds < 1`. I've response like this 
```

nginx_ingress_controller_ingress_upstream_latency_seconds{controller_class="nginx",controller_namespace="kube-system",controller_service="loadbalancer-nginx-ingress-controller-metrics"} | 0.002
nginx_ingress_controller_ingress_upstream_latency_seconds{controller_class="nginx",controller_namespace="kube-system",controller_service="loadbalancer-nginx-ingress-controller-metrics"} | 0.003
nginx_ingress_controller_ingress_upstream_latency_seconds{controller_class="nginx",controller_namespace="kube-system",controller_service="loadbalancer-nginx-ingress-controller-metrics"} | 0.003
```

`0.002s => 2ms`

In a nutshell, I need to use second unit with not rounded responses.

**Comment by tdeekens** [comment]
Got it. I will investigate as soon as I have time. Thanks for the details report.

**Comment by tdeekens** [comment]
Closes and releases as `2.0.1`. Please verify if this works for you :).
